src/Components/TreeView/TreeView.jsx

```diff
@@ -201,7 +201,7 @@ const File = ({
 											className="manage_icon"
 											onClick={() => {
 												onModal('loading');
-												api('GET', `/v1/objects/${containerItem.containerId}/by_id/${objectItem.address.objectId}`, {}, {
+												api('GET', `/v1/objects/${containerItem.containerId}/by_id/${objectItem.objectId}`, {}, {
```

Comment:
As well as this.

Reply:
in new search response we have new format: `{"attributes": {"FileName": "123.svg"}, "objectId": "A1RhLibG9bTRqMszTD1Cp6qsAzSWzqJ2reYmWEFtXxq4"}`